src/NServiceBus.Core.Analyzer.Tests.Common/Sagas/SagaAnalyzerTests.cs

```diff
@@ -1028,6 +1028,115 @@ public Task Handle(StartSaga message, IMessageHandlerContext context)
                 return Assert(source);
             }
         }
+
+        // https://github.com/Particular/NServiceBus/issues/6370
+        [Test]
+        public Task NServiceBusIssue6370PartialClasses()
```

Comment:
Could we use a more descriptive name here?

```suggestion
        public Task SagaHandlersInPartialClasses()
```

or something like that?